po/pt_BR.po

```diff
@@ -25,30 +28,40 @@ msgstr ""
 #: data/io.github.mightycreak.Diffuse.desktop.in:3
 #: data/io.github.mightycreak.Diffuse.appdata.xml.in:4
 msgid "Diffuse"
-msgstr "Diffuse"
+msgstr "Comparador de Arquivos Diffuse"
```

Comment:
change: this should just be "Diffuse" as it is the name of the app, and not a description. It's use as the name of the app in the .desktop file and the AppData. For instamce, if you look for GIMP in GNOME Software, you want to find the app named "GIMP" and not "GIMP the image manipulator" :wink: